### ISSUE TITLE
Daily Bookmarks

### ISSUE THREAD

**Issue by fabiocodeca** [mtopping/obsidian-raindrop#18]
Hi. 
I cannot get the daily bookmarks working.
Any suggestions or ideas on what it should be turned on to make it work?

Thanks

**Comment by cs-mshah** [comment]
Basically, `{{DATE:yyyy-MM-DD}}` doesn't seem to work.

**Comment by erauner12** [comment]
Yeah I could not get it to work either. But, for example this works:
`search: created:>{{DATE:yyyy-MM-DD}}`

**Comment by shaundewberry** [comment]
To get it working, include a "collection:" specification.
e.g.

>\```raindrop
>collection: 0  
>search: created:{{DATE:yyyy-MM-DD}}
>\```

Where {{DATE:yyyy-MM-DD}} is actually a directive for your templating engine to substitute the date.
In my case, I use Templater(https://github.com/SilentVoid13/Templater), so it works like this:

> \#### Today's Bookmarks
>\```raindrop
>collection: 0
>format: table
>search: created:<%tp.date.now("YYYY-MM-DD")%>
>\```

**Comment by cs-mshah** [comment]
Even I use templater, but that snippet doesn't show just today's bookmarks. It shows some previous ones too for me.

**Comment by erauner12** [comment]
> Even I use templater, but that snippet doesn't show just today's bookmarks. It shows some previous ones too for me.

Yeah I've noticed this as well. It does not work how you would expect. Instead, it excludes everything before the provided created date (based on my experience with it). Is anyone else experiencing the same?
